.github/workflows/test.yml

```
@@ -11,9 +11,15 @@ jobs:
     runs-on: 'ubuntu-latest'
     steps:
     - uses: actions/checkout@v4
+      with:
+        submodules: 'true'
     - name: Install dependencies
-      run: sudo apt -y install shellcheck
+      run: |
+        apt-get update
+        apt -y install shellcheck
```

```suggestion
      run: sudo apt-get update && apt-get install -y shellcheck
```

though really we should be using mise-action :)

---

agreed, we should switch to mise actions at some point